Dockerfile

```diff
@@ -72,5 +72,5 @@ COPY --from=builder /usr/sbin/nginx /usr/sbin/nginx
 
 ADD ./rootfs/ /
 
-ENTRYPOINT ["/bin/bash", "-c", "envsubst '${APPLICATION}' < /etc/nginx/conf.d/default.template > /etc/nginx/conf.d/00-default.conf && exec nginx -g 'daemon off;'"]
+ENTRYPOINT ["/bin/bash", "-c", "cat /etc/nginx/conf.d/00-log-format.template /etc/nginx/conf.d/00-default.template > /etc/nginx/conf.d/default.template && envsubst '${APPLICATION}' < /etc/nginx/conf.d/default.template > /etc/nginx/conf.d/00-default.conf && exec nginx -g 'daemon off;'"]
```

Comment:
I believe ` /etc/nginx/conf.d/00-default.template` does not exist (argument of first cat command)

Reply:
It exists: rootfs/etc/nginx/conf.d/default.template → rootfs/etc/nginx/conf.d/00-default.template

Reply:
I see - sorry, I've missed the renaming.

Not so important, but usually numbers are appended to the config files to force an order, setting `00-logging` and `00-default` is not very intuitive. You could leave both without ordering as they are templates, or add some logical ordering. Numbers prefixed to final result are OK to stay

Reply:
As this is getting complex, can we find a better way to do it? Maybe setting all templates to a directory and `cat /path/to/templates/* | envsubst ...`?

Reply:
It seems great. I'll do the change